packages/core/src/card/card.element.ts

```diff
@@ -0,0 +1,53 @@
+import { CdsInternalPanel } from '@cds/core/internal-components/panel';
+import { styles } from './card.element.css.js';
+
+export const CdsCardTagName = 'cds-card';
```

Comment:
The internal components way seems like a better idea to me cause there is single point of truth for the component selector so that's why I decided to go in that direction. We may do the rest for the other components. @mathisscott @coryrylan @Shijir @jeeyun, what do you think?

Reply:
This is something @coryrylan and I have discussed a while back. IIRC, the `const` approach you have here is what we decided on as the way forward in the future. I don't see any issue with switching existing components over to use it if you wanted to take that on in a separate PR.